packages/hot-updater/package.json

```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "@clack/prompts": "^0.9.0",
+    "typescript": "5.8.2",
```

Comment:
I didn't see the usage of this dependency, shouldn't it be in the devDependencies section?

Reply:
here https://github.com/gronxb/hot-updater/blob/08e98fcec7db107bd774acaa760fbc84d0f4c03e/packages/hot-updater/src/utils/transformTsEnv.ts